src/main/java/com/onaio/steps/model/Household.java

```diff
@@ -45,7 +45,9 @@
     public static final String SERVER_STATUS = "Server_Status";
     public static final String UNIQUE_DEVICE_ID = "unique_device_id";
     public static final String ODK_FORM_ID = "odk_form_id";
-    public static final String TABLE_CREATE_QUERY = String.format("CREATE TABLE %s(%s INTEGER PRIMARY KEY, %s TEXT, %s TEXT, %s INTEGER, %s TEXT, %s TEXT, %s TEXT, %s TEXT, %s TEXT, %s TEXT)", TABLE_NAME, ID, NAME, PHONE_NUMBER, SELECTED_MEMBER_ID,STATUS, CREATED_AT ,COMMENTS, UNIQUE_DEVICE_ID, SERVER_STATUS, ODK_FORM_ID);
+    public static final String ODK_JR_FORM_ID = "odk_jr_form_id";
```

Comment:
A question: What does `JR` mean in `ODK_JR...`?

Reply:
JR - > [JavaRosa](https://docs.getodk.org/javarosa/). I guess pertaining to the form_id as part of the standard.

Reply:
Cool :+1: